common/pipeline_storage/src/test/scala/uk/ac/wellcome/pipeline_storage/WorkIndexableTest.scala

```diff
@@ -132,11 +131,11 @@ class WorkIndexableTest
     }
 
     it("overrides a merged work with one that has been merged again") {
-      val mergedWork1 = identifiedWork(nMergedSources = 1)
+      val mergedWork1 = identifiedWork(nSources = 2)
       val mergedWork2 =
-        mergedWork1.copy(state = mergedWork1.state.copy(nMergedSources = 2))
+        mergedWork1.copy(state = mergedWork1.state.copy(nSources = 3))
       val mergedWork3 =
-        mergedWork1.copy(state = mergedWork1.state.copy(nMergedSources = 3))
+        mergedWork1.copy(state = mergedWork1.state.copy(nSources = 4))
```

Comment:
nit: I would name these `mergedWork2, mergedWork3, mergedWork4` so that you the variable name matches the number of sources.